sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/Commands.scala

```diff
@@ -34,7 +34,7 @@ object Commands {
   }
 
   def run(cmd: Seq[String], cwd: File, logger: Logger): Unit = {
-    val toInfoLog = (is: InputStream) => scala.io.Source.fromInputStream(is).getLines.foreach(msg => logger.error(msg))
+    val toInfoLog = (is: InputStream) => scala.io.Source.fromInputStream(is).getLines.foreach(msg => logger.info(msg))
```

Comment:
@cquiroz I’ve also changed this line to log to the `info` logger rather than the `error` logger. I think that logging to the `error` logger was not intentional.

Reply:
I think you’re right, seems like a mistake